docs/pages/management/admin/daemon.mdx

```diff
@@ -2,18 +2,16 @@
 title: Run Teleport as a Daemon
 description: Configure Teleport to run as a daemon using systemd
 ---
+On Linux systems in non-containerized environments, we recommend running the `teleport` binary as a daemon using
+systemd. Using a daemon helps to ensure that the `teleport` process can remain running and available regardless of a controlling terminal session or parent process, and opens up additional
```

Comment:
Consider wrapping these long lines. `reflow-markdown` is a nice VS Code plugin that will do it for you.

Reply:
Great tool! Will be using that from now on